src/services/GooglePlacesService.ts

```diff
@@ -1,183 +1,48 @@
-          lon: restaurantResults.geometry?.location?.lng,
-        };
-
-        this.currRestaurants.set(placeID, restaurant);
-        return Promise.resolve(restaurant);
-      })
-      .catch((err: AxiosError) => {
-        return Promise.reject(err);
-      });
-  }
-
-  // refreshes the invalid placeID, may also result in NOT_FOUND error
-  // refreshing occurs by specifying only the place_id as a field (no other fields allowed)
-  private refreshPlaceID(placeID: string): Promise<string> {
-    return axios
-      .get('https://maps.googleapis.com/maps/api/place/details/json?', {
-        params: {
-          place_id: placeID,
-          fields: 'place_id',
-          key: process.env.REACT_APP_GOOGLE_PLACES_API_KEY,
-        },
-      })
-      .then((response: AxiosResponse) => {
-        const data = response.data;
-
-        if (data?.status === 'NOT_FOUND') {
-          return Promise.reject(new Error('NOT_FOUND ERROR FOR ' + placeID));
-        }
-
-        const newPlaceID = data.result?.place_id;
-        return Promise.resolve(newPlaceID);
+      .get(Routes.RESTAURANTS.RESTAURANT_DETAILS(restaurantId))
+      .then((response: AxiosResponse<Place>) => {
+        const place = response.data;
+        this.currRestaurants.set(restaurantId, place);
+        return Promise.resolve(place);
       })
       .catch((err: AxiosError) => {
         return Promise.reject(err);
       });
   }
 
-  // gets photos for certain placeID
-  public getRestaurantPhoto(photoReference: string): Promise<HTMLImageElement> {
-    return axios
-      .get('https://maps.googleapis.com/maps/api/place/photo?', {
-        params: {
-          photoreference: photoReference,
-          maxheight: 500, //todo: may need to adjust based on FE and restrictions
-          key: process.env.REACT_APP_GOOGLE_PLACES_API_KEY,
-        },
-      })
-      .then((response: AxiosResponse) => {
-        const result: HTMLImageElement = response.data;
-        return Promise.resolve(result);
-      })
-      .catch((err: AxiosError) => {
-        return Promise.reject(err);
-      });
-  }
-
-  // helper function used to refresh placeID and call getRestaurantDetails again for NOT_FOUND error
-  private refreshPlaceIDAndGetDetails(placeID: string): Promise<RestaurantDetails> {
-    return this.refreshPlaceID(placeID)
-      .then((newPlaceID: string) => {
-        const handleNotFoundError = (invalidPlaceID: string) => {
-          return Promise.reject(new Error('NOT_FOUND ERROR FOR ' + invalidPlaceID));
-        };
-        return this.getRestaurantDetails(newPlaceID, handleNotFoundError);
-      })
-      .catch((err: AxiosError) => {
-        return Promise.reject(err);
-      });
+  /**
+   * Gets photos for a restaurant
+   * @param restaurantId the id of the restaurant
+   * */
+  async getRestaurantPhoto(restaurantId: string): Promise<PlacePhoto[]> {
+    const place = await this.getRestaurantDetails(restaurantId);
```

Comment:
Do we want to wrap this in a try/catch or use then/catch blocks in case it fails?